### PR TITLE
Add Taiwan IFRS Account Chart Template --2016/05

### PR DESCRIPTION
**Pull request by jaronemo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by zhangshunrao** [comment]
Don't be too bad papapa good

**Comment by jaronemo** [comment]
#13100  signed the cla

File: addons/l10n_tw_chart/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+# Copyright (C) 2014-now  Jason Wu (jaronemo@msn.com)
+
+{
+    'name': '台灣 國際財務報導準則(IFRS)之會計科(項)目表',
```
Comment (mart-e) [inline addons/l10n_tw_chart/__openerp__.py]: Could you specify a title in English as it is shared among all modules?

File: addons/l10n_tw_chart/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    'version': '1.0',
+    'category': 'Localization/Account Charts',
+    'author': 'Jason Wu',
+    'maintainer': 'Jason Wu(jaronemo@msn.com)',
+    'website': 'http://www.osstw.com',
+    'description': """
+
+    參照  2016 / 05 台灣 國際財務報導準則(IFRS)之會計科(項)目表 更新事項
```
Comment (mart-e) [inline addons/l10n_tw_chart/__openerp__.py]: Having Taiwanese in the description is fine but having a small description for English speakers too is advised.

File: addons/l10n_tw_chart/account_chart_type.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<!--
+
+   # Copyright (C) 2014-now  Jason Wu (jaronemo@msn.com)
```
Comment (mart-e) [inline addons/l10n_tw_chart/account_chart_type.xml]: please do not put copyright notice in each file, marking you as the author is enough

File: addons/l10n_tw_chart/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    """,
+    'depends': ['base', 'account'],
+    'data': [
+
+        'account_chart_type.xml',
+        'account_chart_template.xml',
+        'account_chart_template.yml',
+    ],
+    'license': 'GPL-3',
```
Comment (mart-e) [inline addons/l10n_tw_chart/__openerp__.py]: The license of Odoo is LGPL. Could you remove this line?

File: addons/l10n_tw_chart/i18n/l10n_cn.pot [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Translation of Odoo Server.
```
Comment (mart-e) [inline addons/l10n_tw_chart/i18n/l10n_cn.pot]: remove this file, no reason to have it here

File: addons/l10n_tw_chart/i18n/l10n_tw.pot [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# Translation of Odoo Server.
```
Comment (mart-e) [inline addons/l10n_tw_chart/i18n/l10n_tw.pot]: as the file is empty, no need to keep it too

File: addons/l10n_tw_chart/untitled.xml [hunk 7]
```diff
@@ -0,0 +1,469 @@
+<record id="chart8020" model="account.account.template">
```
Comment (mart-e) [inline addons/l10n_tw_chart/untitled.xml]: missing tag `<?xml version="1.0" encoding="utf-8"?><odoo>` at the beginning of the file

File: addons/l10n_tw_chart/untitled.xml [hunk 8]
```diff
@@ -0,0 +1,469 @@
...
+            <field name="user_type_id" ref="account.data_account_type_direct_costs" />
+            <field name="chart_template_id" ref="l10n_chart_tw_standard_business" />
+        </record>
+        <record id="chart9013" model="account.account.template">
+            <field name="code">9013</field>
+            <field name="name">所得稅費用</field>
+            <field name="reconcile" eval="True" />
+            <field name="user_type_id" ref="account.data_account_type_direct_costs" />
+            <field name="chart_template_id" ref="l10n_chart_tw_standard_business" />
+        </record>
```
Comment (mart-e) [inline addons/l10n_tw_chart/untitled.xml]: missing `</odoo>` at the end

File: addons/l10n_tw_chart/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+    'category': 'Localization/Account Charts',
+    'author': 'Jason Wu',
+    'maintainer': 'Jason Wu(jaronemo@msn.com)',
+    'website': 'http://www.osstw.com',
+    'description': """
+
+    參照  2016 / 05 台灣 國際財務報導準則(IFRS)之會計科(項)目表 更新事項
+
+    """,
+    'depends': ['base', 'account'],
```
Comment (mart-e) [inline addons/l10n_tw_chart/__openerp__.py]: no need for `base`. depending of `account` is enough

**Comment by C3POdoo** [comment]
Dear @jaronemo,

Thank you for your contribution but the version 9.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**